packages/ui/src/bounty/components/BountyPreviewHeader/BountyPreviewHeader.tsx

```diff
@@ -140,20 +159,24 @@ const SuccessfulStageButtons = ({ bounty, activeMember, t }: BountyHeaderButtons
       <ButtonGhost size="large">
         <BellIcon /> {t('common:buttons.notifyAboutChanges')}
       </ButtonGhost>
-      {winner && <ButtonGhost size="large">{t('common:buttons.claimReward')}</ButtonGhost>}
-      {(passed || isContributor) && <ButtonGhost size="large">{t('common:buttons.withdrawStake')}</ButtonGhost>}
+      {winnerConditions && <ClaimRewardButton bountyId={bounty.id} entryId={entryId} reward={new BN(reward)} />}
```

Comment:
Do not call the constructor in the component prop if the component you pass this prop to uses `React.memo`.